integration/docker/csi/alluxio/nodeserver.go

```diff
@@ -139,11 +139,14 @@ func bindMountGlobalMountPointToPodVolPath(req *csi.NodePublishVolumeRequest) (*
 	glog.V(4).Infoln(string(stdoutStderr))
 	if err != nil {
 		if os.IsPermission(err) {
+			glog.V(3).Infof("Permission denied. Failed to run mount bind command. %v", err.Error())
 			return nil, status.Error(codes.PermissionDenied, err.Error())
```

Comment:
Can the error message it self be contained in the status.Error?

Reply:
Are you asking if we can get rid of the error code?